doc/modules/ROOT/pages/gds-session.adoc

```diff
@@ -36,6 +36,17 @@ sessions = GdsSessions(api_credentials=AuraAPICredentials(CLIENT_ID, CLIENT_SECR
 
 === Creating a GDS Session
 
+To create a GDS Session, use the `get_or_create()` method.
+It will create a new session if it does not exist, or connect to an existing one if it does.
+If the session options differ from the existing one, an error is thrown.
+
+The return value of `get_or_create()` is an `AuraGraphDataScience` object.
+It offers a similar API to the `GraphDataScience` object, but it is configured to run on a GDS Session.
+As a convention, always use the variable name `gds` for the return value of `get_or_create()`.
```

Comment:
```suggestion
We recommend to name the variable holding return value of `get_or_create()` `gds`.
```

Reply:
i like the previous one more as it does not have `get_or_create()` and `gds` right next to each other.

I always forget if we write as `we`. I think it was passive. We should have a meeting with Nicola to look over the session docs overall

Reply:
My problem with the first sentence is that it sounds like a requirement.